src/lua/compat.c

```diff
@@ -6,6 +6,24 @@
 
 #include <lua/compat.h>
 
+static const char escaped_slash[] = "\\/";
+
+int
+msgpuck_esc_slash_toggle(struct lua_State *L)
+{
+	assert(lua_isboolean(L, -1) && "boolean argument expected");
+	bool val = lua_toboolean(L, -1);
+	lua_pop(L, 1);
+	lua_pushnumber(L, 0);
+
+	if (val)
+		mp_char2escape['/'] = escaped_slash;
```

Comment:
Just `mp_char2escape['/'] = "\\/";` should be enough.

Reply:
Fixed.